src/main/scala/rps/Main.scala

```diff
@@ -28,7 +29,9 @@ object RPSServer extends App with RouterDerivationModule {
 
   implicit def throwableResponse: ToHttpResponse[Throwable] = null
 
-  val gameRouter = deriveRouter[GameApi](new GameApiImpl)
+  val gameApiImpl = new GameApiImpl()(ec)
```

Comment:
`ec` is implicit, you don't need to pass it explicitly ;)

Reply:
I really need some tips on monday about implicit :3 Seems that there are a non-numerable set of ways where implicit is used, and this causes me lots of headaches. Apparently, implicit was used by Gödel too to demonstrate his incompleteness theorem.